package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "fast-proxy",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
We typically bump version straight on master because otherwise multiple PRs could land in parallel.

Reply:
Totally agree, updating...